core/src/com/theironyard/MyGdxGame.java

```diff
@@ -0,0 +1,123 @@
+package com.theironyard;
+
+import com.badlogic.gdx.*;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.scenes.scene2d.utils.DragAndDrop;
+
+public class MyGdxGame extends ApplicationAdapter {
+    private TextureRegion down, up, right, left;
+	private SpriteBatch batch;
+	private Texture img;
+    String look = "";
+    private float time;
+
+
+
+	private float x, y, xv, yv = 0;
+	public static final float MAX_VELOCITY = 100;
+
+	@Override
+	public void create () {
+		Texture tiles = new Texture("tiles.png");
+		TextureRegion[][] grid = TextureRegion.split(tiles, 16, 16);
+        down = grid[6][0];
+		up = grid[6][1];
+		right = grid[6][3];
+		left = new TextureRegion(right);
+		left.flip(true, false);
+		batch = new SpriteBatch();
+	}
+
+	@Override
+	public void render () {
+        move();
+        TextureRegion img = down;
+        time += Gdx.graphics.getDeltaTime();
+
+        Gdx.gl.glClearColor(0, .6f, 0, 1);
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+
+		batch.begin();
+        batch.draw(img, x, y);
+
+        if (look == "r"){
+            batch.draw(right, x, y);
+        }else if (look == "l"){
```

Comment:
NEVER compare strings with `==`.  This code only works due to a compiler optimization and could change at any time.